client/packages/system/src/Patient/PatientView/AddButton.tsx

```diff
@@ -36,6 +36,11 @@ export const AddButton = () => {
     SplitButtonOption<PatientModal>
   >(options[1] as SplitButtonOption<PatientModal>);
 
+  const onClick = () => {
+    reset();
```

Comment:
the reset is because of the reuse of the modal - otherwise options entered here pollute the other usages